docs/application/vstools/tools/manifest-editor.md

```diff
@@ -421,18 +479,21 @@ For more information on the relationship between the elements, see the [element
 **Expected children:**
 
 | Child element             | Occurrences          |
-| ------------------------- | -------------------- |
+|---------------------------|----------------------|
 | [`privilege`](#privilege) | 1 or more (optional) |
```

Comment:
To keep the consistency of the document, I think we'd better remove links in this chart.

Reply:
Done.